server/api/scraper.py

```diff
@@ -3,74 +3,70 @@
 from selenium.webdriver.chrome.options import Options
 from webdriver_manager.chrome import ChromeDriverManager
 
+def loadChromeDriver():
+    chrome_options = Options()
+    chrome_options.add_argument('--no-sandbox')
+    chrome_options.add_argument('--ignore-certificate-errors')
+    chrome_options.add_argument('--headless')
+    chrome_options.add_argument('--disable-dev-shm-usage')
+    chrome_options.add_argument('--remote-debugging-port=9222')
```

Comment:
Maybe we can pull these out into a constant to make this cleaner. Something like:
```
CHROME_OPTIONS = ['--no-sandbox', etc...]

for option in CHROME_OPTIONS:
    chrome_options.add_argument(option)
```